LimeBean/BeanApi.cs

```diff
@@ -573,6 +573,10 @@ public partial class BeanApi : IDisposable, IBeanCrud, IBeanFinder, IDatabaseAcc
             set { Storage.ConvertEmptyStringToNull = value; }
         }
 
+        /// <summary>
+        /// Gets or Sets whether integers are detected and extracted if in string type variables, when storing to the database.
+        /// Query paramaterisation and the database still handle storage types
+        /// </summary>
         public bool RecognizeIntegers {
```

Comment:
Brilliant, thank you.

Would you agree that some minor work could be done here? Either by only running this conversion on strings if fluid mode is active, or leaving this flag disabled by default?
Right now it's an invisible quirk of the ORM which may appear to be a bug if the user really did want their column to be a string in FluidMode - and in a strongly typed language I would believe their choice of typing over our ability to coerce a value.